llm/app/routers/admin.py

```
@@ -0,0 +1,8 @@
+from fastapi import APIRouter, Depends
+from app.core.security import admin_required
+
+router = APIRouter(prefix="/admin", tags=["admin"])
+
+@router.get("/dashboard")
+def fetch_dashboard(user: dict = Depends(admin_required)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)